client/wfnews-war/src/main/angular/src/app/utils/index.ts

```diff
@@ -755,3 +755,5 @@ return;
     return hc.temp != null;
   });
 }
+
+export const isAndroidViaNavigator = () => navigator.platform.includes('Linux') || navigator.platform.includes('Android');
```

Comment:
We also have a "isAndroidPlatform" in capacitor-services. Might be able to use that property directly